### PR TITLE
Improve README documentation

### PR DESCRIPTION
**Pull request by chambridge**
Format improvements to use rST format that renders properly on GitHub.
Restructuring of file contents.
Addition of Development and Contributing sections.

Easiest viewed by going to the source tree for this branch: https://github.com/quipucords/rho/tree/issues/3

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+rho was written by Adrian Likins <alikins@redhat.com>, Devan Goodwin
```
Comment (noahl) [inline AUTHORS.rst:1]: @jlalbertson should Chris Snyder be in here somewhere?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+All the information that rho needs is stored in the data folder in
+the installed directory. All the auths are stored in the ``credentials``
+file. All the profiles are stored in the ``profiles`` file. The Ansible
+playbook is called ``rho_playbook.yml`` stored in the installed directory.
+The roles created during the scan are stored in the ``roles`` folder. These
+roles are used by ``rho_playbook.yml`` to perform the fact collection.
+
+Running the scan is simple. Just point the rho tool to the profile
+to use, the facts to collect and print the results to a CSV output file.
+Optional parameters are the number of forks Ansible should use and whether
```
Comment (noahl) [inline README.rst]: Consider changing "forks" to "processes"? I think processes is more clear.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+returns that information with every scan, by default.
+
+--------------------------
+Requirements & Assumptions
+--------------------------
+Before installing rho, there are some guidelines about which machine it should be installed on:
+ * rho is written to run on RHEL or Fedora servers.
+ * The machine that rho is installed on must be able to access the machines to be scanned, so it must be on the network and the machines must be running.
+ * The target machines must be running SSH.
+ * The user account that rho uses to SSH into the machine must have adequate permissions to run commands and read certain files. For example, it *cannot* be a /sbin/nologin or /bin/false shell.
```
Comment (noahl) [inline README.rst]: I suggest moving the second sentence of this bullet to the next bullet point.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+``yum install rho``
+
+-----------------------
+Command Syntax & Usage
+-----------------------
+The basic syntax is:
+
+``rho command subcommand [options]``
+
+There are four rho commands:
```
Comment (noahl) [inline README.rst]: "four" -> "eight"

File: doc/command_syntax_usage.rst [hunk 5]
```diff
@@ -0,0 +1,119 @@
+-----------------------
+Command Syntax & Usage
+-----------------------
+The basic syntax is:
+
+``rho command subcommand [options]``
+
+There are four rho commands:
```
Comment (noahl) [inline doc/command_syntax_usage.rst]: s/four/eight/

**Review by noahl — APPROVED** [review]
Some very minor changes, otherwise looks good to me.

I know the errors were there before you reformatted everything, sorry.

File: AUTHORS.rst (hunk 1 above)
Comment (jlalbertson) [reply AUTHORS.rst:1]: Yes - he was part of the development of both rho 1.0 and 2.0, so it would be good to add him in

Comment (chambridge) [reply AUTHORS.rst:1]: added.

File: README.rst (hunk 2 above)
Comment (chambridge) [reply README.rst]: changed.

File: README.rst (hunk 3 above)
Comment (chambridge) [reply README.rst]: moved.

File: README.rst (hunk 4 above)
Comment (chambridge) [reply README.rst]: fixed.

File: doc/command_syntax_usage.rst (hunk 5 above)
Comment (chambridge) [reply doc/command_syntax_usage.rst]: fixed.

**Pull request merged**